### DISCUSSION TITLE
Setting to select visibly devices

### DISCUSSION THREAD

**Discussion by Priskvark** [File-New-Project/EarTrumpet#1434] [Ideas]
This is a program that I've been using for a while. Thank you for your excellent work!

One thing I wish existed is a setting for selecting the audio devices to appear in the list that shows up when clicking on the program's icon. It would be great if we could mark the devices we want to see with, for example, a checkbox and leave the rest unchecked.

In my case, around 40 lines are displayed, but I don't want to see most of these in Eartrumpet. They are only useful in other contexts. For instance, I have many "Audials Sound Capturing" devices and several Bluetooth devices. Of course, most people don't have that many active devices, but there might be others like me who have some extras they don't need or want to see. Having too many options to choose from becomes cluttered and difficult to manage.

Is it possible to implement this in a future version? I understand that you need to assess whether this feature not only suits me but also benefits other users.

**Comment by HydrantHunter** [comment]
I, too, would like this.  For reference, this appears (to me) to be the same as discussion item #1422

**Comment by lilixil** [comment]
It would be very helpful, as you wouldn't have to scroll down the list of devices every time you want to make a change. Optionally, you could add the option to expand a specific device or simply the ability to move devices up/down.
